frontend/src/components/dashboard/streakFlame.css

```
@@ -0,0 +1,46 @@
+.streak-flame-glow.streak-flame-glow--anim{
+  opacity:0.35;
+  animation:streakFlameGlow 1.25s ease-in-out infinite;
+}
+
+.streak-flame{width:54px;height:54px;display:block;transform-origin:50% 70%;}
+
+.streak-flame--anim{
+  animation:streakFlameScale 1.25s ease-in-out infinite;
+}
+
+@keyframes streakFlameScale{
```

Stylelint (17.14.0)

[error] Expected keyframe name "streakFlameScale" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)